src/map/ai/controllers/mob_controller.h

```diff
@@ -88,6 +101,7 @@ class CMobController : public CController
     time_point m_NeutralTime;
     time_point m_WaitTime;
     time_point m_mobHealTime;
+    FOLLOWTYPE m_follow;
```

Comment:
This would be better named as `m_followType`, otherwise it comes off as "I am following this entity", rather than the type of following